org-gtd-delegate.el

```diff
@@ -38,11 +38,16 @@
 
 ;;;; Customization
 
+(defgroup org-gtd-delegate nil
+  "Customize how tasks within the GTD system are delegated."
+  :group 'org-gtd
+  :package-version '(org-gtd . "3.0.0"))
+
 (defcustom org-gtd-delegate-read-func (lambda () (read-string "Who will do this? "))
   "Function that is called to read in the Person the task is delegated to.
 
 Needs to return a string that will be used as the persons name."
-  :group 'org-gtd-organize
+  :group 'org-gtd-delegate
```

Comment:
This may be a correct change but, as indicated somewhere else, this is a business logic change, so it should be suggested separately so we can have a focused conversation on the topic.

Reply:
OK, is there a way I can remove that change from the PR?

Reply:
Raised as issue #157.